packages/account-v2/src/pages/ManualUploadContainer/ManualUploadContainer.tsx

```diff
@@ -5,7 +5,7 @@ import { TManualDocumentTypes } from '../../constants/manualFormConstants';
 import { ManualForm } from '../../containers/ManualForm';
 import { SelfieDocumentUpload } from '../../containers/SelfieDocumentUpload';
 import { useManualForm } from '../../hooks';
-import { OnfidoContainer } from '../../modules/Onfido';
+import { OnfidoContainer } from '../../modules/src/Onfido';
```

Comment:
Is it possible to make any entry point outside `src` directory?

Reply:
I don't get it, Can you elaborate ?

Reply:
Can we exclude `src` directory from the import path?

Reply:
This is case of a package inside another package and we need the src folder to abstract out the package files (e.g. webpack, tailwind etc..). We also need to use these files in the outer package that's why the import path looks like this.